js/apps/admin-ui/src/clients/authorization/ResourceDetails.tsx

```diff
@@ -197,68 +187,30 @@ export default function ResourceDetails() {
             className="keycloak__resource-details__form"
             onSubmit={handleSubmit(submit)}
           >
-            <FormGroup
+            <TextControl
+              name={resourceId ? "owner.name" : ""}
               label={t("owner")}
-              fieldId="owner"
-              labelIcon={
-                <HelpItem helpText={t("ownerHelp")} fieldLabelId="owner" />
-              }
-            >
-              <KeycloakTextInput
-                id="owner"
-                value={client.clientId}
-                isReadOnly
-              />
-            </FormGroup>
-            <FormGroup
+              labelIcon={t("ownerHelp")}
+              defaultValue={client.clientId}
+              readOnly
+            />
```

Comment:
This is not the same as the original where it used the `client.clientId` instead of `owner.name`, is that the same?

Reply:
Yeah, I was surprised with this a bit. I have tested this and it's using "owner.name" on resource creation and "" on edit.